modules/extensions-spring-support/src/test/resources/heisenberg-source-config.xml

```diff
@@ -47,14 +59,20 @@
 
     <flow name="sourceFailedOnRuntime" initialState="stopped">
         <!--invalid initialBatchNumber with negative value -->
-        <heisenberg:listen-payments config-ref="heisenberg" initialBatchNumber="-1"/>
+        <heisenberg:listen-payments config-ref="heisenberg" initialBatchNumber="-1">
+            <heisenberg:success-info knownAddresses="#[[]]"/>
+            <heisenberg:error-info knownAddresses="#[[]]"/>
```

Comment:
@marianogonzalez inline groups that have at least one required parameter are now mandatory as child elements